src/intl/messages.js

```diff
@@ -408,6 +410,8 @@ export const messages: { [messageId: string]: MessageType } = {
     message: 'Loading ...',
     description: 'Pending request to the server indicator label.',
   },
+  localStorage: 'Local',
+  localStorageTooltip: 'Settings are persisted by the browser locally.',
```

Comment:
What is this?
Is it referring to https://github.com/oVirt/ovirt-web-ui/issues/1393? I don't think/sure we need to enable this for all settings options. We discussed that only in the context of the locales. 
WDYT?

Reply:
> Is it referring to #1393?

Yep. It will be (for now) only for language but my plan is to create  Advanced Options section and put it there.
My reasons:
1. adding this switch/checkbox next to language dropdown looks awkward
2. we are adding more options that require opt-out i.e. console options and notifications. Adding a switch/checkbox for every property would look bad.

Reply:
I'm not sure that opt-out for console options and notification are really needed. I can't think of a reason for this but ok.